tests/functional/test_bucket.py

```diff
@@ -6,22 +6,23 @@
 from botocore.exceptions import ClientError
 from dotenv import load_dotenv
 
-from ...bucket import upload_directory, upload_image_get_metadata
-from ...settings import ROOT_DIR
+from IIIFIngest.bucket import upload_directory, upload_image_get_metadata
+
+TESTS_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+IMAGES_DIR = os.path.join(TESTS_DIR, "images")
```

Comment:
If you're running these with `pytest`, would these be available as a fixture from the `conftest.py`?